src/constants.zig

```diff
@@ -105,20 +105,24 @@ pub const vsr_releases_max = config.cluster.vsr_releases_max;
 
 /// The maximum cumulative size of a final TigerBeetle output binary - including potential past
 /// releases and metadata.
-pub const multiversion_binary_platform_size_max = blk: {
+pub fn multiversion_binary_platform_size_max(options: struct { macos: bool, debug: bool }) u64 {
```

Comment:
Open to other ways of doing this; not sure if we want a `fn` in constants.zig.

Reply:
Hm, I need to look how this whole thing is setup, but what gives me a pause is the larger issue: `release.zig` is build orchestration, it _shouldn't_ (transitively) depend on anything in the main binary. 

So, what's strange is not that we add something to constants, but that release.zig pulls in multiversioning.zig which pulls in IO and constants.

This is not necessary bad for any particular reason, but still makes me uneasy.

Reply:
> So, what's strange is not that we add something to constants, but that release.zig pulls in multiversioning.zig which pulls in IO and constants.

Right, so the release logic uses the `Header` struct, some misc `Release` structs and the `parse_*`. Those mostly have no dependencies, except for `constants.vsr_releases_max`.

Could pull them out into a standalone file, and include that both from `release.zig` and `multiversioning.zig`.